website/index.html

```diff
@@ -0,0 +1,440 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+
+    <title>Indiepen</title>
+    <meta
+      name="description"
+      content="An independent and privacy-friendly solution to present your code example to the people."
+    />
+    <meta name="HandheldFriendly" content="True" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="shortcut icon" href="/website/favicon.png" />
+    <link
+      rel="stylesheet"
+      href="/website/styles.css"
+      media="none"
+      onload="if(media!='all')media='all'"
+    />
+    <noscript>
+      <link rel="stylesheet" href="/website/styles.css" />
+    </noscript>
+
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      rel="preload"
+      href="https://fonts.googleapis.com/css2?family=Inter:wght@400;700;900&family=Source+Code+Pro:wght@500&display=swap"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Inter:wght@400;700;900&family=Source+Code+Pro:wght@500&display=swap"
+      media="print"
+      onload="this.media='all'"
+      rel="stylesheet"
+    />
```

Comment:
We should probably use npm packages here and bundle the fonts with snowpack, so we don't have to make requests to Google.